Resources/config/default_settings.yml

```diff
@@ -1,9 +1,4 @@
 parameters:
-    ez_recommendation.default.yoochoose.customer_id: ~
-    ez_recommendation.default.yoochoose.license_key: ~
```

Comment:
Not in the `default_settings.yml` anymore, it should be defined in `config.yml` by user.